package/gluon-core/files/lib/netifd/proto/gluon_wired.sh

```diff
@@ -7,15 +7,29 @@ init_proto "$@"
 proto_gluon_wired_init_config() {
         proto_config_add_boolean transitive
         proto_config_add_int index
+        proto_config_add_boolean add_vxlan_layer
```

Comment:
Please move the addition and handling of this option to a separate commit. I think a shorter name like `vxlan` would look better.

The option would probably not exist for long, as non-VXLAN setups should be phased out eventually (but we don't have a schedule for that yet - will depend on the patches mentioned in #2155 I guess).